### PR TITLE
Add blog design and disqus comments support

### PR DESCRIPTION
**Pull request by faustinoaq**
screenshot_20171109_164217

@amberframework/contributors

### CODE REVIEW

File: src/views/blog/show.slang [hunk 1]
```diff
@@ -1,8 +1,16 @@
 - @title = "#{post["title"]} - Blog - Amber Framework"
+header.header.header-inverse.overflow-hidden.orange-header
+  .container.text-center
+    .row
+      .col-12.col-lg-8.offset-lg-2
+        h1 = post["title"]
+        p.fs-20.opacity-70
+          = post["date"]
+          | by
+          = post["author"]
```
Comment (marksiemers) [inline src/views/blog/show.slang]: Can this be a shared partial?

Comment (faustinoaq) [reply src/views/blog/show.slang]: No needed anymore, I removed duplication and added author link style, to avoid blue link with orange background.

**Comment by eliasjpr** [comment]
Can we add disquss and make the blog post similar to this style  
http://thetheme.io/thesaas/blog-single.html?

**Comment by faustinoaq** [comment]
@eliasjpr Yeah, let me try 😄

**Comment by faustinoaq** [comment]
screenshot_20171109_210109

@eliasjpr 👆  TheSaaS blog theme looks beautiful 😄 

I tried to modify guides too but the afix sidebar is a headache 😅 Maybe we should use Gitbook for documentation and write some guides on official page using an style like http://kemalcr.com/cookbook/hello_world/ or https://elixir-lang.org/getting-started/introduction.html

**Comment by faustinoaq** [comment]
screenshot_20171109_213132

Adding Disqus comments 😄

**Comment by faustinoaq** [comment]
Hi @elorest I got the photos via https://unsplash.com/ (no copyright) and https://crystal-lang.org/blog/ (maybe copyright?)

> If you ever need any more astrophotography pictures I have a ton lol.

I think we can use your photos here instead of curent ones, Please suggest us 3 photos for these post and one for the [the next one](https://github.com/amberframework/amberframework.org/pull/58)

Also use https://tinypng.com/ to reduce the size as well as possible 😄

**Comment by faustinoaq** [comment]
star

GutHub star buton added! 😄

**Comment by marksiemers** [comment]
Do we have any concern (if we are using a new photo per blog post) of that blowing up the size of the repo? Or there isn't enough volume to matter?

Eventually, they should be hosted somewhere, and potentially with a CDN, right?

Right now, its probably too early to worry about.

**Comment by faustinoaq** [comment]
@marksiemers You are right, I gonna take your advice using an external image hosting.

**Pull request merged**